deploy/fabfile.py

```diff
@@ -6,15 +6,15 @@
 
 Staging hook:
 
-  forever start -l $HOME/hookshot.log -a deploy/hookshot.js -p 3000 -b staging -c "cd $HOME/staging/current && git pull && bundle exec jekyll build --config _config.yml,_deploy_config.yml >> $HOME/hookshot.log"
-  forever restart deploy/hookshot.js -p 3000 -b staging -c "cd $HOME/staging/current && git pull && bundle exec jekyll build --config _config.yml,_deploy_config.yml >> $HOME/hookshot.log"
-  forever stop deploy/hookshot.js -p 3000 -b staging -c "cd $HOME/staging/current && git pull && bundle exec jekyll build --config _config.yml,_deploy_config.yml >> $HOME/hookshot.log"
+  forever start -l $HOME/githooked.log -a deploy/githooked.js -p 3000 -b staging -c "cd $HOME/staging/current && git pull && bundle exec jekyll build --config _config.yml,_deploy_config.yml >> $HOME/githooked.log"
+  forever restart deploy/githooked.js -p 3000 -b staging -c "cd $HOME/staging/current && git pull && bundle exec jekyll build --config _config.yml,_deploy_config.yml >> $HOME/githooked.log"
+  forever stop deploy/githooked.js -p 3000 -b staging -c "cd $HOME/staging/current && git pull && bundle exec jekyll build --config _config.yml,_deploy_config.yml >> $HOME/githooked.log"
 
 Production hook:
 
-  forever start -l $HOME/hookshot.log -a deploy/hookshot.js -p 4000 -b production -c "cd $HOME/production/current && git pull && bundle exec jekyll build --config _config.yml,_deploy_config.yml >> $HOME/hookshot.log"
-  forever restart deploy/hookshot.js -p 4000 -b production -c "cd $HOME/production/current && git pull && bundle exec jekyll build --config _config.yml,_deploy_config.yml >> $HOME/hookshot.log"
-  forever stop deploy/hookshot.js -p 4000 -b production -c "cd $HOME/production/current && git pull && bundle exec jekyll build --config _config.yml,_deploy_config.yml >> $HOME/hookshot.log"
+  forever start -l $HOME/githooked.log -a deploy/githooked.js -p 4000 -b production -c "cd $HOME/production/current && git pull && bundle exec jekyll build --config _config.yml,_deploy_config.yml >> $HOME/githooked.log"
+  forever restart deploy/githooked.js -p 4000 -b production -c "cd $HOME/production/current && git pull && bundle exec jekyll build --config _config.yml,_deploy_config.yml >> $HOME/githooked.log"
+  forever stop deploy/githooked.js -p 4000 -b production -c "cd $HOME/production/current && git pull && bundle exec jekyll build --config _config.yml,_deploy_config.yml >> $HOME/githooked.log"
```

Comment:
For these and the staging hooks, would you mind switching the log file back to `hookshot.log`? We have a log rotator that I think watches for "hookshot.log" @mbland is that correct?